### ISSUE TITLE
Code build failing running on ubuntu

### ISSUE THREAD

**Issue by java-blip** [haneenmahd/swudo#1]
i have listed `type: "module"` in package.json.

Still it gave an error and shows some logs
It showed the following log :
```
> swudo@1.0.0 build /home/runner/work/swudo/swudo
> tsc && npm run fix:path


> swudo@1.0.0 fix:path /home/runner/work/swudo/swudo
> node scripts/fixPaths.js

/home/runner/work/swudo/swudo/scripts/fixPaths.js:1
import fs from 'fs';
       ^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! swudo@1.0.0 fix:path: `node scripts/fixPaths.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the swudo@1.0.0 fix:path script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-05-17T12_45_48_379Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! swudo@1.0.0 build: `tsc && npm run fix:path`
npm ERR! Exit status 1
npm ERR! 
```

**Comment by haneenmahd** [comment]
Yeah! I am really working on the issue.

I think we should add 'scripts' directory to package.json:
https://github.com/haneenmahd/swudo/blob/fd61e3715b3c06c3e3778bed6ac79407ed7accfc/package.json#L13-L15

**Comment by haneenmahd** [comment]
The issue has been fixed on the commit `b62580cceded5adbe95d421920c03723c4834853`
